mongodb/vibe/db/mongo/settings.d

```diff
@@ -341,6 +342,10 @@ class MongoClientSettings
 	string sslPEMKeyFile;
 	string sslCAFile;
 	MongoAuthMechanism authMechanism;
+	/// Flag to send handshake to MongoDB servers to identify capabilities. May not work on old MongoDB versions.
+	bool nextgen = true;
```

Comment:
camelCase and I think `nextGen` is a bad name.
`supportsIdentification` or `supportsCapabilitiesCheck`, ...

Reply:
actually removed this now (can send the identify handshake in any version)